### PR TITLE
docs: improve grammar and wording on CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Toxicable**
Duplicate of: #16300

Related: https://github.com/angular/material2/pull/4230#pullrequestreview-34435184
Credit to @fxck for making the original changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
 
 ## <a name="question"></a> Got a Question or Problem?
 
-Please, do not open issues for the general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [Stack Overflow](https://stackoverflow.com/questions/tagged/angular) where the questions should be tagged with tag `angular`.
+Do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](https://stackoverflow.com/questions/tagged/angular) where the questions should be tagged with tag `angular`.
```
Comment (gkalpak) [inline CONTRIBUTING.md]: Why `Stack Overflow` --> `StackOverflow`? I thought the official version was as two words.

Comment (Toxicable) [inline CONTRIBUTING.md]: Sounds about right to me, fixed.

**Comment by petebacondarwin** [comment]
This needs root approval @IgorMinar, etc.

**Comment by petebacondarwin** [comment]
@Toxicable - the commit message target should not be `aio`. This is a general docs commit, so the message should be simply:

```
docs: improve grammar and wording on CONTRIBUTING.md
```

(Note the correct spelling of grammar :-) )

**Comment by Toxicable** [comment]
Oh gotcha, gimme a few

**Comment by Toxicable** [comment]
@petebacondarwin Done

**Review by IgorMinar — APPROVED** [review]

**Pull request merged**